tests/tests/test_regressions.py

```diff
@@ -5,14 +5,23 @@
 import requests
 
 from .scenario import Scenario
-from .services import API_URL
+from .services import API_URL, EDITOAST_URL
 
 REGRESSION_TESTS_DATA_FOLDER = Path(__file__).parent / "regression_tests_data"
 REGRESSION_TESTS_JSON_FILES = [json_file.name for json_file in REGRESSION_TESTS_DATA_FOLDER.resolve().glob("*.json")]
 
 
-def _pathfinding_with_payload(base_url, payload, infra_id, accept_400):
+def _load_infra(editoast_url, infra_id, accept_400):
+    r = requests.post(editoast_url + f"infra/{infra_id}/load")
+    if r.status_code // 100 != 2:
+        if r.status_code // 100 == 4 and accept_400:
+            return None
+        raise RuntimeError(f"Infra load {r.status_code}: {r.content}")
+
+
+def _pathfinding_with_payload(base_url, editoast_url, payload, infra_id, accept_400):
     payload["infra"] = infra_id
+    _load_infra(editoast_url, infra_id, accept_400)
```

Comment:
You can move this outside of the pathfinding function, it should be called before and not inside